File: C#/Ex.2.cs
using System;

class Program {
  public static void Main (string[] args) {
    int[] numero = new int[4];
    
    Console.WriteLine("Insira um número: ");
    numero[0] = Convert.ToInt32(Console.ReadLine());
    
    for(int n = 0; n < 3; n++){
      numero[n + 1] = numero[n] + 1;
    }
    for(int n = 0; n < 4; n++){
      Console.Write($"[{numero[n]}]");
    }
  }
}

File: Python/Ex.2.py
numero = [0,0,0,0]

for n in range(0, 4):
  if n == 0:
    numero[n] = int(input("Insira um valor: "))
  else:
    if n > 0:
      numero[n] = numero[n - 1] * 7

print(numero)